superset-frontend/src/SqlLab/components/AceEditorWrapper/index.tsx

```diff
@@ -22,14 +22,15 @@ import 'brace/mode/sql';
 import 'brace/theme/github';
 import 'brace/ext/language_tools';
 import ace from 'brace';
-import { areArraysShallowEqual } from '../../reduxUtils';
-import sqlKeywords from '../utils/sqlKeywords';
+import { areArraysShallowEqual } from 'src/reduxUtils';
+import sqlKeywords from '../../utils/sqlKeywords';
```

Comment:
This could use the src alias too, if it helps avoid future shufflin'
```suggestion
import sqlKeywords from 'src/SqlLab/utils/sqlKeywords';
```